src/Stores/User/actions.js

```diff
@@ -0,0 +1,68 @@
+/* @flow */
+import { login, signUp } from '../../utils/api';
+
+import { saveToken } from '../../utils/tokenStorage';
+
+export const ActionTypes = {
+  USER_LOGIN: 'USER_LOGIN',
+  USER_LOGIN_FAILURE: 'USER_LOGIN_FAILURE',
+  USER_LOGIN_SUCCESS: 'USER_LOGIN_SUCCESS',
+};
+
+const loginUser = () => ({ type: ActionTypes.USER_LOGIN });
+
+const loginUserSuccess = (data: { email: string }) => ({
+  data,
+  type: ActionTypes.USER_LOGIN_SUCCESS,
+});
+
+const loginUserFailure = (error: *) => ({
+  error,
+  type: ActionTypes.USER_LOGIN_FAILURE,
+});
+
+export const loginUserThunk = (email: string, password: string) => (
+  dispatch: Dispatch
+) => {
+  dispatch(loginUser());
+
+  return login(email, password).then(resp => {
+    if (resp.error) {
+      loginUserFailure(resp.error.message);
+      return;
```

Comment:
would this make sense here?

```
return loginUserFailure(resp.error.message);
```

instead of doing an empty return? Which I think are a bit ugly :)

Reply:
the eslint rule `consistent-return` does not like that way.

Think we need to rethink how to handle all of these cases

Reply:
the eslint fault was
```
[eslint] Expected to return a value at the end of arrow function. (consistent-return)
```

which just means there needs to be a return on success as well

Reply:
I'll fix it